unittests/PhasarLLVM/DataFlowSolver/IfdsIde/EdgeFunctionComposerTest.cpp

```diff
@@ -16,9 +16,9 @@ static unsigned CurrAddTwoEfId = 0;
 struct MyEFC : EdgeFunctionComposer<int> {
   MyEFC(std::shared_ptr<EdgeFunction<int>> F,
         std::shared_ptr<EdgeFunction<int>> G)
-      : EdgeFunctionComposer<int>(std::move(F), std::move(G)){};
+      : EdgeFunctionComposer<int>(F, G){};
   std::shared_ptr<EdgeFunction<int>>
```

Comment:
```suggestion
      : EdgeFunctionComposer<int>(F, G){};

  std::shared_ptr<EdgeFunction<int>>
```
An empty line here would make it more readable ;)

PS: Why did you remove the `moves`?